File: Practice/21/Python/21.py
def bmi(weight: float, height: float) -> float:
    return weight / (height*height)
def print_bmi(bmi: float) -> float:
    if bmi < 18.0:
        print("Underweight")
    elif bmi >= 18.0 and bmi < 25.0:
        print("Normal")
    elif bmi >= 25.0 and bmi < 30.0:
         print("Overweight")
    elif bmi > 30:
        print("Obesity")

weight, height = map(int, input().split())
height /= 100
print_bmi(bmi(weight, height))

File: Practice/21/C++/21/21/21.cpp
#include<iostream>
using namespace std;

double BMI(double weight, double height)
{
	return weight /pow(height, 2);
}
void printBMI(double BMI)
{
	if (BMI < 18.5)
		cout << "Underweight";
	else if (BMI >= 18.5 && BMI < 25.0)
		cout << "Normal";
	else if (BMI >= 25.0 && BMI < 30)
		cout << "Overweight";
	else if (BMI >= 30.0)
		cout << "Obesity";
}

int main()
{
	double weight, height;
	cin >> weight >> height;
	height /= 100;
	printBMI(BMI(weight, height));
}